messages/teleporter/message_manager_test.go

```diff
@@ -154,18 +152,33 @@ func TestShouldSendMessage(t *testing.T) {
 	}
 	for _, test := range testCases {
 		t.Run(test.name, func(t *testing.T) {
+			validTeleporterMessage.DestinationChainID, err = getChainIDBytes(test.destinationChainID.String())
```

Comment:
I think we can just do `validTeleporterMessage.DestinationChainID = test.destinationChainID`, since they're both `[32]byte`. We might have to do some casts, but should be lightweight.

Reply:
fixed